tools/create-cluster

```diff
@@ -53,7 +54,7 @@ platform:
     network:
       if: tt0
 publish: External
-pullSecret: '$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/openshift-pull-secret -H "Metadata-Flavor: Google")'
+pullSecret: $(echo \'"${PULL_SECRET}"\')
```

Comment:
@crawford here, this is the change that is already in the image in openshift-gce-devel-ci - I updated it to test my release PR, too early-but didn't think it would be an issue bc the libvirt template/setup is broken as/is anyways.

Reply:
Thanks! That makes much more sense now.

nit: `”${PULL_SECRET}”` (no need for the subshell)